packages/gatsby-cli/src/reporter/reporter.ts

```diff
@@ -83,18 +83,23 @@ class Reporter {
   /**
    * Log arguments and exit process with status 1.
    */
-  panic = (errorMeta: ErrorMeta, error?: Error | Array<Error>): never => {
-    const reporterError = this.error(errorMeta, error)
+  panic = (
+    errorMeta: ErrorMeta,
+    error?: Error | Array<Error>,
+    pluginName?: string
```

Comment:
Would it make sense to make 3rd arg an "options" object in case we ever want to add other kind of metadata that could be used without `pluginName`? 
```suggestion
    options?: {
      pluginName?: string
    }
```

Reply:
We could certainly do that, but what options would that be? Any information we want to pass can already happen inside the first arg of the structured error (the free-form `context` field).